src/advanced-settings/AdvancedSettings.test.jsx

```diff
@@ -61,7 +61,10 @@ describe('<AdvancedSettings />', () => {
     const { getByText } = render(<RootWrapper />);
     await waitFor(() => {
       expect(getByText(messages.headingSubtitle.defaultMessage)).toBeInTheDocument();
-      expect(getByText(messages.headingTitle.defaultMessage)).toBeInTheDocument();
+      const advancedSettingsElement = getByText('Advanced settings', {
```

Comment:
Why did you remove `messages.headingTitle.defaultMessage`?

Reply:
It`s my mistake, thanks
Good catch 👍